ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -338,6 +370,14 @@ Apply Custom Manifest in DataScienceCluster CustomResource Using Test Variables
          END
     END
 
+Apply DevFlags in DataScienceCluster CustomResource
+    [Documentation]    Apply devFlags to a DSC file
+    Log To Console    Applying devFlags
+    ${file_path} =    Set Variable    tasks/Resources/Files/
+    FOR    ${cmp}    IN    @{COMPONENT_LIST}
+        Run     sed -i'' -e "s|<${cmp}_devflags>||g" ${file_path}dsc_apply.yml
```

Comment:
so, is it actually removing the dev flags? 
Also, isn't this the same action done at https://github.com/red-hat-data-services/ods-ci/pull/1762/files#diff-11c2f02f8419b8b4ee8bf5f117645cfe5da320a349f21813d8da399c47a7ffb3R369

Reply:
No, it is setting the key as empty as it is by default